src/components/ui/Utilities/formatDate.ts

```diff
@@ -4,25 +4,32 @@
 import memoize from 'fast-memoize';
 
 export interface DateOptions {
-  weekday?: string;
-  year?: string;
-  month?: string;
-  day?: string;
+  weekday?: 'long' | 'short' | 'narrow';
+  year?: 'numeric' | '2-digit';
+  month?: 'long' | 'short' | 'narrow' | 'numeric' | '2-digit';
+  day?: 'numeric' | '2-digit';
 }
```

Comment:
Strictly speaking this is a change in our published types. Some customers might have to fix their type definitions.

This is OK for two reasons:

* It only narrows the expected types to what they should have been. Most customers will not be affected.
* Any customer who supplied different values and gets a build error as a result of this change **will have just discovered an actual bug**.